src/main.rs

```diff
@@ -591,34 +608,98 @@ fn get_setup(args: &[String]) -> Setup {
         }
     };
 
-    if matches.opt_present(HELP) {
+    let opt_present = |opt| matches.opt_present(opt) || env_var_present(opt);
```

Comment:
Sure can do.

Reply:
That will require a lot more code changes since `matches` is local to `get_setup` that will turn `opt_present(FOO)` and `opt_str(FOO)` into `opt_present(matches, FOO)` and `opt_str(matches, FOO)`.

Reply:
Yeah I see how that would get clunky. Let's keep it as is then. Is the code free of clippy warnings?

Reply:
it also seems unnecessary to turn a closure that's not used in any other function into a standalone function. That's kinda what lambdas are for. I can see making it another function if it were used elsewhere but it's not.

Reply:
> 

I will triple check.

Reply:
Yep. It's all good. No clippy warnings.